integrations/hardhat/test/fixture-projects/hardhat-project/hardhat.config.ts

```diff
@@ -0,0 +1,11 @@
+// We load the plugin here.
+import { HardhatUserConfig } from 'hardhat/types';
+
+import '../../../index.js';
```

Comment:
try linking the dependency using a `link:` or `workspace` directive in the `package.json`. that'll give you something closer to what you'll get in production

Reply:
Did you suggest I should add a package.json inside this fixture-project?

It seems to be a common practice to directly import the plugin file:

[hardhat-ethers](https://github.com/NomicFoundation/hardhat/blob/main/packages/hardhat-ethers/test/fixture-projects/hardhat-project/hardhat.config.js#L1), 
[hardhat-ganache](https://github.com/NomicFoundation/hardhat/blob/main/packages/hardhat-ganache/test/fixture-projects/hardhat-project/hardhat.config.ts#L1)
[hardhat-gas-trackooor](https://github.com/d3or/hardhat-gas-trackooor/blob/main/test/hardhat-waffle-project/hardhat.config.ts#L6)